.env.example

```diff
@@ -1,2 +1,5 @@
-MONGO_URI=<mongo uri>
+MONGO_USERNAME=
+MONGO_PASSWORD=
+MONGO_HOST=mongo
+MONGO_PORT=27017

```

Comment:
I think we need to add an optional MONGO_DB here to allow specifying the database to use at the connected host

Reply:
Yes you are right, I was relying on the default one (which is `test` generated automatically), but the DB name should be configurable.